modules/tensor_mechanics/src/materials/ADLAROMANCEStressUpdateBase.C

```diff
@@ -692,10 +697,11 @@ ADLAROMANCEStressUpdateBase::convertOutput(const std::vector<Real> & old_input_v
     return 0.0;
 
   ADReal expout = std::exp(rom_output);
-  if (expout > 1.0e-10)
-    expout -= 1.000000E-10;
+  if (expout > _rom_strain_output_cutoff)
+    expout -= _rom_strain_output_cutoff;
```

Comment:
I don't understand whats going on here. Let's say `expout` is 4*`_rom_strain_output_cutoff`. Then you subtract it once, but it is still larger than the cutoff value... ?

Reply:
Yeah...I'm not sure why it's like this either. This is a part of the way LAROMANCE does things and how the ROMs are formulated...

Reply:
Is there a change here that's desired? A general comment string that we're only checking for low bounds? Or is okay as is?